oap-server/server-starter/src/main/assembly/application.yml

```diff
@@ -59,6 +60,7 @@ storage:
 #    bulkSize: ${SW_STORAGE_ES_BULK_SIZE:20} # flush the bulk every 20mb
 #    flushInterval: ${SW_STORAGE_ES_FLUSH_INTERVAL:10} # flush the bulk every 10 seconds whatever the number of requests
 #    concurrentRequests: ${SW_STORAGE_ES_CONCURRENT_REQUESTS:2} # the number of concurrent requests
+#  mysql:
```

Comment:
Here is a mysql module config, but no parameter. Please confirm about it.

Reply:
According to https://github.com/apache/incubator-skywalking/pull/1964 pr,mysql config would be placed in datasource-settings.properties file.Also,I'll make a fit for the file as soon as possible about the feature that override configuration through environment variables.

Reply:
Yes. MySQL is based on the old setting file style, which is different. :) We could choose to move them into application.yml. 

Anyway, that is another feature to discuss.

Reply:
yep,if move them into ` application.yml` , the setting file style will appear to be more uniform.:P